### DISCUSSION TITLE
Reduce bug bounty page congestion

### DISCUSSION THREAD

**Discussion by Abbasjafri-syed** [solodit/feedback#27] [New Feature Requests]
Bug bounty page is very congested making it difficult to navigate. It will be better if bounties row is streched to the whole page with filter applied at the top..like the same design of Immunefi...

This design implementation for findings page will also be appreciated as it enhances experience of searching issues more conveniently.

![Capture](https://github.com/solodit/feedback/assets/101616115/470a9d31-d661-4212-9f24-d7020bf0c7b3)

**Comment by hans-cyfrin** [comment]
Thanks for the suggestion. We will discuss and try improve the user experience with your feedback in mind.

**Comment by hans-cyfrin** [comment]
With the new feature of folding the search filter, I believe this issue is resolved now. 
Thanks for the feedback.
